team/team.go

```diff
@@ -58,9 +60,20 @@ func Save(team Team, fluidkeysDirectory string) error {
 	}
 
 	rosterFilename := filepath.Join(rosterDirectory, "roster.toml")
+	signatureFilename := rosterFilename + ".asc"
+
+	signature, err := signingKey.MakeArmoredDetachedSignature(roster.Bytes())
+	if err != nil {
+		return fmt.Errorf("failed to sign team roster: %v", err)
+	}
+
 	if err = atomic.WriteFile(rosterFilename, roster); err != nil {
 		return fmt.Errorf("failed write team roster: %v", err)
 	}
+	err = atomic.WriteFile(signatureFilename, bytes.NewBufferString(signature))
```

Comment:
Would `[]byte(signature)` work instead of `bytes.NewBufferString(signature)` ?

Reply:
Sadly not...
```
cannot use ([]byte)(signature) (type []byte) as type io.Reader in argument to "github.com/fluidkeys/fluidkeys/vendor/github.com/natefinch/atomic".WriteFile:
	[]byte does not implement io.Reader (missing Read method)
```